content/docs/command-reference/pprint.md

```diff
@@ -55,23 +55,29 @@ $ mlem pprint rf
 ## Example: Showing remote data
 
 ```cli
-$ mlem pprint https://github.com/iterative/example-mlem-get-started/iris.csv
-⏳️ Loading meta from https://github.com/iterative/example-mlem-get-started/tree/main/.mlem/dataset/iris.csv.mlem
+$ mlem pprint https://raw.githubusercontent.com/iterative/example-mlem-get-started/4-pack/.mlem/data/iris.csv
+⏳️ Loading meta from https://raw.githubusercontent.com/iterative/example-mlem-get-started/4-pack/.mlem/data/iris.csv.mlem
 {'artifacts': {'data': {'hash': '45109f850511f9474665f2c26f4c79f3',
                         'size': 2470,
-                        'type': 'dvc',
                         'uri': 'iris.csv'}},
- 'object_type': 'dataset',
- 'reader': {'dataset_type': {'columns': ['sepal length (cm)',
-                                         'sepal width (cm)',
-                                         'petal length (cm)',
-                                         'petal width (cm)'],
-                             'dtypes': ['float64',
-                                        'float64',
-                                        'float64',
-                                        'float64'],
-                             'index_cols': [],
-                             'type': 'dataframe'},
+ 'data_type': {'columns': ['sepal length (cm)',
+                           'sepal width (cm)',
+                           'petal length (cm)',
+                           'petal width (cm)'],
+               'dtypes': ['float64', 'float64', 'float64', 'float64'],
+               'index_cols': [],
+               'type': 'dataframe'},
+ 'object_type': 'data',
```

Comment:
It looks like datasets are now stored in a different directory (`/.mlem/data/` instead of `/.mlem/dataset/`).

```diff
-'object_type': 'dataset',
+'object_type': 'data,
```

I'm curious, but does the `object_type` set this directory?